tide/core/data/TiffPyramidReader.cpp

```diff
@@ -68,6 +68,7 @@ struct TiffPyramidReader::Impl
 TiffPyramidReader::TiffPyramidReader(const QString& uri)
     : _impl{new Impl{uri}}
 {
+    TIFFSetWarningHandler(0);
```

Comment:
This should be done statically. If 0 is a pointer, it should be nullptr. And we don't use a custom handler for these messages in the end?